cisco_aci/datadog_checks/cisco_aci/tags.py

```diff
@@ -111,18 +158,28 @@ def tenant_mapper(self, edpt):
             self.log.debug('adding eth level tags: %s' % eth_meta)
         return tags
 
-    def get_tags(self, obj, obj_type):
-        tags = []
-        if obj_type == 'endpoint_group':
-            tags = self.tenant_mapper(obj)
-        if obj_type == 'tenant':
-            tags = ["tenant:" + obj]
-        if obj_type == 'application':
-            tags = self.app_tags(obj)
-        return tags
+    # def get_tags(self, obj, obj_type):
+    #     # TODO I am really not a big fan of this method because it's combining 3 behaviors that are not related
```

Comment:
this is a relic from when the three things were less well separated in the code itself, probably good if we can remove it as long as we're sure they all work